docs/user-guide/api-mediation/api-mediation-multi-tenancy.md

```diff
@@ -0,0 +1,289 @@
+        "zowe.apiml.gateway"
+],
+                "serviceId": "gateway"
+            }
+        ]
+    },
+    {
+        "apimlId": "apiml2",
+        "services": [
+            {
+                "status": "UP",
+                "customMetadata": {},
+                "apiId": [
+        "zowe.apiml.gateway"
+],
+                "serviceId": "gateway"
+            }
+        ]
+]
+```
+
+## Validate successful configuration
+
+Use the `/registry` endpoint to validate successful configuration. The response should contain all API ML domains represented by `apimlId`, and information about onboarded services.
+
+## Troubleshooting
+
+
+### ZWESG100W  
+
+Cannot receive information about services on API Gateway with apimlId 'Gateway-CA32' because: Received fatal alert: certificate_unknown; nested exception is javax.net.ssl.SSLHandshakeException: Received fatal alert: certificate_unknown
+
+**Reason**  
+The trust between the domain and the Cloud Gateway was not established. 
+
+**Action**  
+Review your certificate configuration.
+
+### No debug messages similar to Gateway-CA32 completed with onComplete are produced
+
+ **Reason**  
+ Domain Gateway is not correctly onboarded to Discovery Service in Central API ML. 
+ 
+ **Action**  
+ Review Gateway static definition. Check the Central Discovery Service dashboard if the domain Gateway is displayed. 
+
+## Onboarding domain cloud-gateway service to central discovery service
+
+The central Cloud Gateway can onboard Cloud Gateways of all domains. This can be achieved similar to additional registrations of the Gateway:
+
+- Dynamic configuration via zowe.yaml
+- Dynamic configuration via Environment variables
+
+### Configurations
+
+#### Dynamic configuration: YML
+
+Users must set the following property for the domain cloud-gateway to dynamically onboard to central Discovery Service.
+
+`components.cloud-gateway.apiml.service.additionalRegistration`
+
+Use the following example as a reference on how to set the value for this property in zowe.yml.
+
+**Example:**
+```
+components.cloud-gateway.apiml.service.additionalRegistration:
+    # central API ML (in HA, for non-HA mode use only 1 hostname)
+       - discoveryServiceUrls:      https://ca32.lvn.broadcom.net:27554/eureka/,https://ca32.lvn.broadcom.net:37554/eureka/
+ 	    routes:
+              - gatewayUrl: /
+                serviceUrl: /
+```
+
+#### Dynamic configuration: Environment variables
+
+The list of additional registrations is extracted from environment variables. You can define a list of objects by following YML->Environment translation rules. 
+
+The previous example can be substituted with the following variables:
+
+```
+ZWE_CONFIGS_APIML_SERVICE_ADDITIONALREGISTRATION_0_DISCOVERYSERVICEURLS=https://ca32.lvn.broadcom.net:27554/eureka/,https://ca32.lvn.broadcom.net:37554/eureka/
+ZWE_CONFIGS_APIML_SERVICE_ADDITIONALREGISTRATION_0_ROUTES_0_GATEWAYURL=/
+ZWE_CONFIGS_APIML_SERVICE_ADDITIONALREGISTRATION_0_ROUTES_0_SERVICEURL=/
+```
```

Comment:
Can we add a section that describes what the variables specify after this example?

Reply:
I added  a two sentences and will ask someone to take a look.

Reply:
It seems to me pretty clear that these `additionalRegistration*` variables are the way how to register domain gateway to one or more central gateways. Without it, the registration will not happen. I am not sure what else should be documented here.